src/core/Notification/Notification/Notification.tsx

```diff
@@ -0,0 +1,154 @@
+import React, { Component, forwardRef } from 'react';
+import { default as styled } from 'styled-components';
+import classnames from 'classnames';
+import {
+  HtmlDiv,
+  HtmlDivWithRef,
+  HtmlButton,
+  HtmlButtonProps,
+} from '../../../reset';
+import { Icon } from '../../Icon/Icon';
+import { Heading } from '../../Heading/Heading';
+
+import {
+  BaseNotificationProps,
+  notificationClassNames,
+  baseClassName,
+} from '../BaseNotification/BaseNotification';
+import { AutoId } from '../../utils/AutoId/AutoId';
+import { getConditionalAriaProp } from '../../../utils/aria';
+import { SuomifiThemeProp, SuomifiThemeConsumer } from '../../theme';
+import { baseStyles } from './Notification.baseStyles';
+
+export interface NotificationProps extends BaseNotificationProps {
+  /** Text to label the close button. Visible + `aria-label` in regular size and only used as `aria-label` in small screen variant */
+  closeText: string;
+  /** Click handler for the close button */
+  onCloseButtonClick?: () => void;
+  /** Custom props passed to the close button */
+  closeButtonProps?: Omit<HtmlButtonProps, 'onClick' | 'aria-label'>;
+  /** Use small screen styling */
+  smallScreen?: boolean;
+  /** LabelText is header of component */
+  labelText: string;
+}
+
+interface InnerRef {
+  forwardedRef: React.RefObject<HTMLDivElement>;
+}
+
+class BaseNotification extends Component<NotificationProps & InnerRef> {
+  render() {
+    const {
+      className,
+      ariaLiveMode = 'assertive',
+      status = 'neutral',
+      labelText,
+      children,
+      id,
+      actionElements,
+      closeText,
+      onCloseButtonClick,
+      smallScreen,
+      closeButtonProps = {},
+      ...passProps
+    } = this.props;
+
+    const {
+      className: customCloseButtonClassName,
+      'aria-describedby': closeButtonPropsAriaDescribedBy,
+      ...closeButtonPassProps
+    } = closeButtonProps;
+    const variantIcon =
+      status === 'neutral'
+        ? 'info'
+        : status === 'success'
+        ? 'checkCircle'
+        : 'error';
+    return (
+      <HtmlDivWithRef
+        as="section"
+        {...passProps}
+        className={classnames(baseClassName, className, {
+          [`${baseClassName}--${status}`]: !!status,
+          [notificationClassNames.smallScreen]: !!smallScreen,
+        })}
+      >
+        <HtmlDiv className={notificationClassNames.styleWrapper}>
+          <HtmlDiv className={notificationClassNames.iconWrapper}>
+            <Icon icon={variantIcon} className={notificationClassNames.icon} />
+          </HtmlDiv>
+
+          <HtmlDiv
+            className={notificationClassNames.textContentWrapper}
+            id={id}
+            aria-live={ariaLiveMode}
+          >
+            <HtmlDiv className={notificationClassNames.content}>
+              {labelText && (
+                <Heading variant="h2" className={notificationClassNames.label}>
```

Comment:
Could or should the heading level be an optional prop so that the users could control the heading level they want to use? What do you think?

Reply:
If it is optional, it should be limited to `h2-h6`? So that it cannot be `h1` at all.

Reply:
Now it has props: headingVariant?: 'h2' | 'h3' | 'h4' | 'h5';
there was no option in heading for h6...

Reply:
Rather use hLevels found in HtmlH.tsx.

Reply:
Fixed. now using hLevels.